definitions/pipelines/bisulfite.cwl

```diff
@@ -90,15 +96,16 @@ steps:
         in:
             vcf: pileup/vcf
             reference: reference_index
+            assay_non_cpg_sites: assay_non_cpg_sites
         out:
-            [cpgs,cpg_bedgraph]
+            [methylation_bed,methylation_bedgraph]
     bedgraph_to_bigwig:
         run: ../tools/bedgraph_to_bigwig.cwl
         in:
-            bedgraph: vcf2bed/cpg_bedgraph
+            methylation_bedgraph: vcf2bed/methylation_bedgraph
             reference_sizes: reference_sizes
         out:
-            [cpg_bigwig]
+            [methylation_bigwigs]
```

Comment:
why are some arrays plural (methylation_bigwigs) and some singular (methylation_bedgraph)?  Again, not a big deal, but being consistent would make the code more readable.

Reply:
Changed it. Made everything singular as most of them were singular.